backend/main.py

```
@@ -6,20 +6,33 @@
 import dotenv
 from pydantic import BaseModel
 import uvicorn
+from typing import Annotated, List
 
 from fastapi import APIRouter, Depends, HTTPException
 from sqlalchemy.orm import Session
 import models
 import schemas
 from utils import verify_token
 
+from fastapi import Depends, FastAPI, HTTPException, status
```

Ruff (0.8.2)

Redefinition of unused `Depends` from line 18

Remove definition: `Depends`

(F811)